PureLayout/PureLayout/PureLayoutDefines.h

```diff
@@ -34,10 +34,18 @@
 #define __PureLayout_MinBaseSDK_iOS_8_0                   (TARGET_OS_IPHONE && __IPHONE_OS_VERSION_MAX_ALLOWED > __IPHONE_7_1)
 #define __PureLayout_MinBaseSDK_OSX_10_10                 (!TARGET_OS_IPHONE && __MAC_OS_X_VERSION_MAX_ALLOWED > __MAC_10_9)
 
+#ifdef __MAC_10_11
+#define __PureLayout_MinBaseSDK_OSX_10_11                 (!TARGET_OS_IPHONE && __MAC_OS_X_VERSION_MAX_ALLOWED >= __MAC_10_11)
```

Comment:
While that does work, I'd personally prefer to follow [Apple's recommendation](https://developer.apple.com/library/ios/documentation/DeveloperTools/Conceptual/cross_development/Using/using.html#//apple_ref/doc/uid/20002000-1114741-CJADDEIB) in [Availability.h](http://www.opensource.apple.com/source/CarbonHeaders/CarbonHeaders-18.1/Availability.h):

```
    It is also possible to use the *_VERSION_MIN_REQUIRED in source code to make one
    source base that can be compiled to target a range of OS versions.  It is best
    to not use the _MAC_* and __IPHONE_* macros for comparisons, but rather their values.
    That is because you might get compiled on an old OS that does not define a later
    OS version macro, and in the C preprocessor undefined values evaluate to zero
    in expresssions, which could cause the #if expression to evaluate in an unexpected
    way.
```

Would you mind making that change? (I see that we are also violating this rule in code on `master`, and I'd be happy to clean that up in a subsequent PR.)